app/controllers/admin/service_orders_controller.rb

```diff
@@ -3,9 +3,9 @@ class Admin::ServiceOrdersController < Admin::AdminController
   layout 'admin'
 
   def index
-    @service = ServiceOrder.order("id ASC")
+    @service = ServiceOrder.order("id ASC").page params[:page]
```

Comment:
these are very basic thing in rails, other members please check this note above. thanks!
@tendousouji @tranquangvu @Tranxuanvu :+1: